blocks/timeline/index.js

```diff
@@ -28,6 +28,7 @@ registerBlockType( 'rtgb/timeline', {
 				className: 'timeline-title',
 				placeholder: __( 'Enter title' ),
 				tagName: 'h4',
+				formattingControls: [ '' ],
```

Comment:
Is it because we do not want to show toolbar?

Reply:
Yes, If we use `inlineToolbar: false` then it will stick to block level. But no option to completely remove it.

Reply:
Then I think it should be `formattingControls: []` , in place of `formattingControls: [ '' ]`

Reply:
Updated.